src/code_80040E50.c

```diff
@@ -328,26 +329,27 @@ void func_80041480(s16 *arg0, s16 arg1, s16 arg2, s16 *arg3) {
     }
 }
 
-s32 func_80041500(Vec3f arg0, f32 arg1, f32 arg2, f32 arg3) {
+s32 vec3f_set_dupe_2_electric_boogaloo(Vec3f arg0, f32 arg1, f32 arg2, f32 arg3) {
     arg0[0] = arg1;
     arg0[1] = arg2;
     arg0[2] = arg3;
     return &arg0;
 }
 
-#ifdef MIPS_TO_C
-//generated by mips_to_c commit 3c3b0cede1a99430bfd3edf8d385802b94f91307
-// weird diff
+// This causes the sqrtf call to be compiled down to the MIPS sqrt.s command
+// rather than truly calling the sqrtf function via `jal sqrtf`
+// func_80041530 WILL NOT MATCH if this is removed
+#pragma intrinsic (sqrtf)
```

Comment:
This can go at the top of the file. No comment necessary imo. (we do this in math_utils.c for the same reason)